Update privacy data handling details for dbt Wizard (#9491)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/wizard-platform-privacy-data.md

</IntroText>

<Expandable alt_header="Does dbt Wizard access my warehouse data?">
  

> <Constant name="wizard" /> can run dbt commands and queries on your behalf. Every query needs your explicit permission first. When a query runs, d<Constant name="wizard" /> sends the results &mdsah; which may include row-level data &mdsah; to the LLM provider so it can respond in your session. 

Reviewer: ```suggestion
<Constant name="wizard" /> can run dbt commands and queries on your behalf. Every query needs your explicit permission first. When a query runs, <Constant name="wizard" /> sends the results &mdash; which may include row-level data &mdash; to the LLM provider so it can respond in your session. 
```

> The provider doesn't keep this data or use it for training, under a zero data retention (ZDR) agreement. <Constant name="wizard" /> doesn't export or store your raw warehouse data outside the <Constant name="dbt_platform" />. We recommend to always review AI output for accuracy.

Reviewer: ```suggestion
The provider doesn't keep this data or use it for training, under a zero data retention (ZDR) agreement. <Constant name="wizard" /> doesn't export or store your raw warehouse data outside the <Constant name="dbt_platform" />. We recommend you always review AI output for accuracy.
```

<Expandable alt_header="Does dbt Wizard access my warehouse data?">

> <Constant name="wizard" /> can run dbt commands and queries on your behalf, and every query needs your explicit permission first. When a query runs, <Constant name="wizard" /> sends those results &mdsah; which may include row-level data &mdsah; to the AI provider so it can respond in your session. 

Reviewer: ```suggestion
<Constant name="wizard" /> can run dbt commands and queries on your behalf, and every query needs your explicit permission first. When a query runs, <Constant name="wizard" /> sends those results &mdsah; which may include row-level data &mdash; to the AI provider so it can respond in your session. 
```
